share/functions/realpath.fish

```diff
@@ -6,8 +6,10 @@
 # If we see that there is an external command by that name we just return. That will cause fish to
 # run the external command. On the other hand, if an external command isn't found we define a
 # function that will provide fallback behavior.
-if not type -q -P realpath
-    function realpath --description 'fallback realpath implementation'
-        builtin fish_realpath $argv[-1]
-    end
+if not command -v realpath > /dev/null
+	if command -v grealpath > /dev/null
+		alias realpath=grealpath
+	else
+		alias realpath=fish_realpath
```

Comment:
I'm ambivalent about this aspect of the proposed change. Since nearly all invocations specify only a single path, with no flags, this specific change won't matter in practice. On the other hand the extremely small number of invocations which might pass a flag (in particular `-P` or `--physical`) will be broken since our builtin doesn't recognize any flags.